diff -ur a/functions b/functions
--- a/functions	2018-06-09 01:49:20.225392563 +0200
+++ b/functions	2018-06-09 01:49:45.197757010 +0200
@@ -650,7 +650,7 @@
 
     local workdir= kernver=$1 arch=$(uname -m) buildroot
 
-    if ! workdir=$(mktemp -d --tmpdir mkinitcpio.XXXXXX); then
+    if ! workdir=$(mktemp -d -t mkinitcpio.XXXXXX); then
         error 'Failed to create temporary working directory in %s' "${TMPDIR:-/tmp}"
         return 1
     fi
diff -ur a/lsinitcpio b/lsinitcpio
--- a/lsinitcpio	2018-06-09 01:49:20.225392563 +0200
+++ b/lsinitcpio	2018-06-09 01:49:55.222035685 +0200
@@ -144,7 +144,7 @@
     local -a binaries explicitmod modules foundhooks hooks
     local kernver ratio columns image=$1
 
-    workdir=$(mktemp -d --tmpdir lsinitcpio.XXXXXX)
+    workdir=$(mktemp -d -t lsinitcpio.XXXXXX)
     trap 'rm -rf "$workdir"' EXIT
 
     columns=80
